### ISSUE TITLE
divide by zero in demoinfocs-golang/common.DemoHeader.FrameTime

### ISSUE THREAD

**Issue by marksamman** [markus-wa/demoinfocs-golang#108]
**Describe the bug**
Running the code below on the demo file linked results in a panic (runtime error) because it's dividing by zero in common.DemoHeader.FrameTime.

**To Reproduce**
https://www.dropbox.com/s/z0xjroygkdec6l6/742301.dem?dl=0

Code:
```go
		f, err := os.Open("742301.dem")
		if err != nil {
			panic(err)
		}
		defer f.Close()

		p := dem.NewParser(f)
		p.RegisterEventHandler(func(e events.SayText2) {
			 p.CurrentTime()
		})
		p.ParseToEnd()
```

**Library version**
v1.1.0

**Comment by markus-wa** [comment]
Yikes.

This looks like another case of info being missing in the demo header of some demos.

Unfortunately I don't have access to a PC the next 3 weeks so it'll have to wait.

The fix will probably be to return 0 if PlaybackTime is 0

**Comment by markus-wa** [comment]
If you could create a PR with the fix I'd be able to merge it though 🙂

**Comment by marksamman** [comment]
Submitted a pull request: #109. Any chance you could tag a release after merging? 🙂

**Comment by markus-wa** [comment]
> Any chance you could tag a release after merging?

Yes sir

**Comment by markus-wa** [comment]
Thanks for the fix, I've tagged it with v1.1.1 ✌🏻
